fuel/transformers/__init__.py

```diff
@@ -8,6 +9,39 @@
 from fuel import config
 from fuel.streams import AbstractDataStream
 from fuel.schemes import BatchSizeScheme
+from ..exceptions import AxisLabelsMismatchError
+
+log = logging.getLogger(__name__)
+
+
+class ExpectsAxisLabels(object):
+    """Mixin used to verify axis labels.
+
+    Notes
+    -----
+    Provides a method :meth:`verify_axis_labels` that should be called
+    with the expected and actual values for an axis labels tuple. If
+    `actual` is `None`, a warning is logged; if it is non-`None` and does
+    not match `expected`, a :class:`AxisLabelsMismatchError` is raised.
```

Comment:
Do you think we should eventually make this behaviour configurable? For instance, the user might want to be strict and raise an error if `axis_labels` is `None`, or to log a warning in case of a mismatch instead of an error.